Singularity

```diff
@@ -0,0 +1,150 @@
+# Generated by Neurodocker version 0.4.3-15-gad62560
+# Timestamp: 2019-04-15 12:05:03 UTC
+# 
+# Thank you for using Neurodocker. If you discover any issues
+# or ways to improve this software, please submit an issue or
+# pull request on our GitHub repository:
+# 
+#     https://github.com/kaczmarj/neurodocker
+
+Bootstrap: docker
+From: neurodebian:stretch-non-free
+
+%post
+export ND_ENTRYPOINT="/neurodocker/startup.sh"
+apt-get update -qq
+apt-get install -y -q --no-install-recommends \
+    apt-utils \
+    bzip2 \
+    ca-certificates \
+    curl \
+    locales \
+    unzip
+apt-get clean
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen
+dpkg-reconfigure --frontend=noninteractive locales
+update-locale LANG="en_US.UTF-8"
+chmod 777 /opt && chmod a+s /opt
+mkdir -p /neurodocker
+if [ ! -f "$ND_ENTRYPOINT" ]; then
+  echo '#!/usr/bin/env bash' >> "$ND_ENTRYPOINT"
+  echo 'set -e' >> "$ND_ENTRYPOINT"
+  echo 'export USER="${USER:=`whoami`}"' >> "$ND_ENTRYPOINT"
+  echo 'if [ -n "$1" ]; then "$@"; else /usr/bin/env bash; fi' >> "$ND_ENTRYPOINT";
+fi
+chmod -R 777 /neurodocker && chmod a+s /neurodocker
+
+apt-get update -qq
+apt-get install -y -q --no-install-recommends \
+    gcc \
+    g++
+apt-get clean
+rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+test "$(getent passwd neuro)" || useradd --no-user-group --create-home --shell /bin/bash neuro
+su - neuro
+
+export PATH="/opt/miniconda-latest/bin:$PATH"
+echo "Downloading Miniconda installer ..."
+conda_installer="/tmp/miniconda.sh"
+curl -fsSL --retry 5 -o "$conda_installer" https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh
+bash "$conda_installer" -b -p /opt/miniconda-latest
+rm -f "$conda_installer"
+conda update -yq -nbase conda
+conda config --system --prepend channels conda-forge
+conda config --system --set auto_update_conda false
+conda config --system --set show_channel_urls true
+sync && conda clean --all && sync
+conda create -y -q --name neuro_py36
+conda install -y -q --name neuro_py36 \
+    'python=3.6'
+sync && conda clean --all && sync
+bash -c "source activate neuro_py36
+  pip install --no-cache-dir  \
+      'ipython' \
+      'nipype==1.1.5' \
```

Comment:
looks like the singularity file was not regenerated.

Reply:
Jop, my bad, sorry. The singularity related lines wrt file generation and building were commented back then and uncommented in the last commit. The latest commit addresses this, including updated files.